app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_with model: task do |form| %>
```

Comment:
Why do you add simple_form but don't use it?

Reply:
This task should not contain this code, so we can ignore this problem